src/supremm/account.py

```diff
@@ -178,8 +178,8 @@ class DbAcct(Accounting):
     Helper class to get job records from the store
     """
 
-    def __init__(self, resource_id, conf):
-        super(DbAcct, self).__init__(resource_id, conf)
+    def __init__(self, resconf, conf):
```

Comment:
Please can you explain why this changed? It looks like the only use of resconf is to pull in the resource_id (which is what was being used before).

Reply:
Resolved in https://github.com/ubccr/supremm/pull/259/commits/7a424e68517909a53032843e1673c6808030f269 and https://github.com/ubccr/supremm/pull/259/commits/9224350ea211d934aca42b8e496fce0162763b7c